src/pages/courts/[courtID]/index.tsx

```diff
@@ -19,7 +19,7 @@ const CourtDetail = () => {
   useEffect(() => {
     const fetchCourts = async () => {
       const { data } = await api(apiUrl as string, { method: "get" });
```

Comment:
As you developed a courtsApi, it that possible for you to rebuild this fetch function through your new api? So that we may get rid of the axios package in cc-admin.

Reply:
No matter how much I like to use native fetch, it is hard to discard Axios. It is a controversial topic.

Reply:
Normally, it is a good practice to be consistent if we use the rtk query. I have tied to use the rkt query -getCourtById query to fetch one court data and transform the court data here, but I got some errors, such as the date. So I still keep using Axios here. And fetching one court data is not included in this ticket card.